exonum-java-binding-core/rust/src/proxy/service.rs

```diff
@@ -151,26 +151,32 @@ impl Service for ServiceProxy {
         }
     }
 
-    fn public_api_handler(&self, context: &ApiContext) -> Option<Box<Handler>> {
-        let node = NodeContext::new(
-            self.exec.clone(),
-            context.blockchain().clone(),
-            *context.public_key(),
-            context.node_channel().clone(),
+    fn wire_api(&self, builder: &mut ServiceApiBuilder) {
+        let self_ = self.clone();
+        builder.public_scope().endpoint(
+            "",
```

Comment:
Would you please clarify what "" is? Does this method have any unintended side-effects (e.g., creates a new handler that processes (?) each request)?

**CC** @alekseysidorov